### ISSUE TITLE
release-4.13: GDCM CMP0042 warning

### ISSUE THREAD

**Issue by thewtex** [InsightSoftwareConsortium/ITK#493]
As [described on Discourse](https://discourse.itk.org/t/patches-for-itk-4-13-2/1582/3),

```
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   gdcmcharls
   gdcmjpeg12
   gdcmjpeg16
   gdcmjpeg8
   gdcmsocketxx
   gdcmuuid

This warning is for project developers.  Use -Wno-dev to suppress it.
-- Generating done
```

**Comment by thewtex** [comment]
@blowekamp is working on an update to GDCM -- if this does not address the issue, we should set the CMake policy as required.

**Comment by blowekamp** [comment]
resolved in GDCM with malaterre/GDCM#83 then in ITK by #485
